kolibri/core/device/api.py

```diff
@@ -113,6 +113,22 @@ def get(self, request, format=None):
         info["python_version"] = "{major}.{minor}.{micro}".format(
             major=version_info.major, minor=version_info.minor, micro=version_info.micro
         )
+
+        if not request.user.is_superuser:
+            # If user is not superuser, return just free space available and kolibri version
+            keys_to_remove = [
+                "urls",
+                "database_path",
+                "device_id",
+                "os",
```

Comment:
Note that `device_id` (`instance_id`) and `os` are in [/api/public/info/](https://kolibri-demo.learningequality.org/api/public/info/), as they are needed for identification (and showing OS icons) with peer discovery. So we don't currently treat those as private info. But those are accessed through a different endpoint, so this won't affect it.